import_processor/readme/INSTALL.rst

```diff
@@ -0,0 +1,3 @@
+To use this module, you have to install the ``jsonpath-ng`` python librairy.
+
+``pip3 install jsonpath-ng``
```

Comment:
jsonpath-ng and openpyxl are optional features as stated in the source and exception. Please remove this file